qiskit/backends/ibmq/ibmqjob.py

```diff
@@ -237,38 +225,18 @@ def done(self):
         Note behavior is slightly different than Future objects which would
         also return true if successfully cancelled.
         """
-        return self.status['status'] == JobStatus.DONE
+        return self.status == JobStatus.DONE
 
     @property
     def cancelled(self):
         return self._cancelled
 
-    @property
-    def exception(self):
```

Comment:
Again, the exception method is meant to comply with concurrent.futures API as mentioned above. Since local jobs do actually use futures which have this available then it's nice if ibmq jobs have the same interface.

Reply:
This is something we have to review as well, can't find a reason strong enough in favor of copying Futures API against sacrificing simplicity.

Reply:
I agree. If there's a compelling reason to move away from futures then we should. Let's discuss it further perhaps in a new issue.